Makefile

```diff
@@ -334,6 +341,27 @@ input_cleanup: ## deletes the secret/CM, used by the services as input
 	oc kustomize ${OUT}/${NAMESPACE}/input | oc delete --ignore-not-found=true -f -
 	${CLEANUP_DIR_CMD} ${OUT}/${NAMESPACE}/input
 
+##@ CRC BMO SETUP
+.PHONY: crc_bmo_setup
+crc_bmo_setup:
```

Comment:
As openstack-baremetal-operator is automatically depoloyed by openstack-operator today I suggest to include this pre-requisite installation target in `openstack_prep` target.

Reply:
I have kept it specific to crc as openshift installations come with the operator.

Reply:
We normally don't do the crc stuff like `crc_storage` in openstack_prep or openstack_deploy_prep. However, I have added to run it with openstack_prep. Let me know if you've better ideas.